packages/priority-queue/src/index.js

```diff
@@ -143,8 +150,15 @@ export const createQueue = () => {
 		return true;
 	};
 
+	const reset = () => {
+		waitingList = [];
+		elementsMap = new WeakMap();
+		isRunning = false;
+	};
```

Comment:
Some documentation would be good, regardless of whether inline in code or in the README.md.

To be honest, I'm not really clear on its purpose either. Do we not need a flush? Is a separate `element` argument of `add` sufficient? A new instance of `createQueue`?

Reply:
The purpose is to empty the waiting list (not run any callback waiting)

Reply:
So these would not work?

>Is a separate element argument of add sufficient? A new instance of createQueue?

Reply:
No, I don't think so. I already use separate instance for each "add". That argument serves to override the previous callbacks but it won't clear the queue.